erts/emulator/beam/erl_bif_guard.c

```diff
@@ -19,7 +19,12 @@
  */
 
 /*
- * Numeric guard BIFs.
+ * This file implements the former GC BIFs. They used to do a GC when
+ * they needed heap space. Because of changes to the implementation of
+ * literals, those BIFs are now allowed to allocate heap fragments
+ * (using HeapFragOnlyAlloc()). Not that they must NOT call HAlloc(),
```

Comment:
"Note that they must NOT" (missing e from Note)

Reply:
Fixed.